components/service/glean/src/main/java/mozilla/components/service/glean/CommonMetricData.kt

```diff
@@ -74,4 +74,20 @@ interface CommonMetricData {
         val filteredNames = sendInPings.filter { it != DEFAULT_STORAGE_NAME }
         return filteredNames + defaultStorageDestinations
     }
+
+    /**
+     * Helper function to return the key name used in the metric storage that works with
+     * empty category names
+     *
+     * @return The key name used with this metric when storing and retrieving it
+     */
+    fun getKeyName(): String {
```

Comment:
Let's make this a property as discussed [here](https://github.com/mozilla-mobile/android-components/pull/1985#discussion_r254986941).